packages/experimental/RadioGroup/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@fluentui-react-native/experimental-radio-group",
+  "version": "0.1.0",
+  "description": "A cross-platform RadioGroup component using the Fluent Design System. Currently only implemented in win32.",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui-react-native.git",
+    "directory": "packages/experimental/RadioGroup"
+  },
+  "dependencies": {
+    "@uifabricshared/foundation-compose": "^1.12.1",
+    "@fluentui-react-native/adapters": ">=0.9.0 <1.0.0",
+    "@fluentui-react-native/experimental-activity-indicator": "^0.6.0",
+    "@fluentui-react-native/experimental-shadow": "0.2.10",
```

Comment:
Do you use these packages in your package?

Reply:
Nope, accidentally had these left in. Removed!